src/instructlab/train/linux_train.py

```diff
@@ -237,16 +238,21 @@ def model_generate(user):
     output_dir = "./training_results"
     per_device_train_batch_size = 1
 
+    training_args = {}
+    training_args["per_device_train_batch_size"] = per_device_train_batch_size
+    training_args["fp16"] = use_fp16
+    training_args["bf16"] = not use_fp16
```

Comment:
The bf16 issue is addressed in #993

Reply:
Sounds good, but this PR isn't really intended to deal with any specific training options, the point is to allow the advanced user to override any of them without needing to make changes to the code.